pkg/apb/provision.go

```diff
@@ -13,7 +13,7 @@ import (
 // Maybe apb defines its own interface and accepts that optionally
```

Comment:
Do we also need the context for bind and unbind? Right now bind does an ```oc logs <podname>```. If that pod is not in the default namespace, bind won't work.